data/beaconrestapi/src/integration-test/java/tech/pegasys/teku/beaconrestapi/AbstractBeaconRestAPIIntegrationTest.java

```diff
@@ -46,7 +47,11 @@
 public abstract class AbstractBeaconRestAPIIntegrationTest {
   static final okhttp3.MediaType JSON = okhttp3.MediaType.parse("application/json; charset=utf-8");
   static final TekuConfiguration config =
-      TekuConfiguration.builder().setRestApiPort(0).setRestApiDocsEnabled(false).build();
+      TekuConfiguration.builder()
+          .setRestApiPort(0)
+          .setRestApiDocsEnabled(false)
+          .setRestApiHostWhitelist(List.of("127.0.0.1", "localhost"))
+          .build();
```

Comment:
we could actually have another test alongside these that creates a config with different elements int he whitelist to show it working for real if we wanted to...